ligo-webide-frontend/base-components/packages/workspace/src/components/GistUploadModals.js

```diff
@@ -147,8 +147,10 @@ export default class GistUploadModals extends PureComponent {
       >
         {this.state.gistLink === '' && <DebouncedFormGroup
           ref={this.input}
-          label={<div>To upload your project you need to add github token</div>}
-          placeholder='Github token'
+          label={<div>To upload your project you need to add github token, or
+                      leave the default to create a gist without a Github account.
+                </div>}
+          initialValue={atob('Z2hwX3dNYkNNS2Z1MGs1d1loZzl4aDRVODBlT1BBdUpGUjF6b3Z4TA==')}
```

Comment:
Where did you take `initialValue`? You should put it to the `this.state.token` as an initial value instead of `''`

Reply:
> Where did you take `initialValue`?

`DebouncedFormGroup` contains a `DebouncedInput` which has an `initialValue` field.

> You should put it to the `this.state.token` as an initial value instead of `''`.

Ah makes sense, I fixed it.